v22.2/create-changefeed.md

```diff
@@ -28,8 +34,20 @@ To create a changefeed, the user must be a member of the `admin` role or have th
 Parameter | Description
 ----------|------------
 `table_name` | The name of the table (or tables in a comma separated list) to create a changefeed for.<br><br>**Note:** Before creating a changefeed, consider the number of changefeeds versus the number of tables to include in a single changefeed. Each scenario can have an impact on total memory usage or changefeed performance. See [Create and Configure Changefeeds](create-and-configure-changefeeds.html) for more detail.
-`sink` | The location of the configurable sink. The scheme of the URI indicates the type. For more information, see [Sink URI](#sink-uri) below.
-`option` / `value` | For a list of available options and their values, see [Options](#options) below.
+`sink` | The location of the configurable sink. The scheme of the URI indicates the type. For more information, see [Sink URI](#sink-uri).
+`option` / `value` | For a list of available options and their values, see [Options](#options).
+
+### CDC transformation parameters
+
+The [Change Data Capture Transformations](cdc-transformations.html) page provides detail on the functionality, syntax, and use cases for changefeeds created with transformations. 
```

Comment:
nit: recommend a single sentence explaining what a CDC transformation is, before linking out to it. What you have from `cdc-transformations.md` would be fine!

> Change data capture transformations allow you to define the change data emitted to your sink when you create a changefeed. See the [Change Data Capture Transformations](cdc-transformations.html) page for detail on the functionality, syntax, and use cases for changefeeds created with transformations.